### PR TITLE
UTOPIA-1181: (frontend) create missing UI

### PR DESCRIPTION
**Pull request by Bottle7**
# Description

This PR includes the following proposed change(s):

- Program Area can accept accountability and review PIAs

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Updating Testing Framework(s)
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/utils/object-modification.util.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ export const modifyNestedObject = <T>(
   // return the value if path is empty; last key
   if (!path) return value;
 
+  console.log('prevState', prevState);
+  console.log('path', path);
```
Comment (binggaofw) [inline src/frontend/src/utils/object-modification.util.ts:13]: can delete console.log statements

**Comment by kushal-arora-fw** [comment]
@binggaofw  - comments addressed.

**Review by kushal-arora-fw — COMMENTED** [review]
Overall, I kinda feel that the usage of so many different versions of Review form is bizarre. Should have been one common component used across.

**Review by binggaofw — APPROVED** [review]
look good to me

File: src/frontend/src/pages/PIAForm/index.tsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ const PIAFormPage = () => {
     key: keyof IPiaForm,
     isEager?: boolean,
   ) => {
-    // DO NOT allow state changes in the view mode
+    // DO NOT allow state changes in the view mode unless it is a review page
     if (mode === 'view' && !pathname?.split('/').includes('review')) return;
```
Comment (antsand) [inline src/frontend/src/pages/PIAForm/index.tsx:112]: I know this is not part of this ticket, but i somehow do not like this type of logic. seems very ugly.

**Review by antsand — APPROVED** [review]

File: src/frontend/src/pages/PIAForm/index.tsx (hunk 2 above)
Comment (kushal-arora-fw) [reply src/frontend/src/pages/PIAForm/index.tsx:112]: 100%

**Pull request merged**